GOVERNANCE.md

```diff
@@ -156,40 +187,41 @@ backgrounds, viewpoints and talents in our team, which is why we explicitly do
 not define code as the sole metric on which council membership will be
 evaluated.
 
-If a Council member becomes inactive in the project for a period of one year,
+Council membership is assigned for a two year period, with no limit on how many
+periods can be served.
+
+Council members can renounce at any time and are
+encouraged to do so if they foresee they won't be able to attend their
+responsibilities for an extended interval of time.
+
+If a Council member becomes inactive in the project for a period of six months,
 they will be considered for removal from the Council. Before removal, inactive
 Member will be approached by the BDFL to see if they plan on returning to
 active participation. If not they will be removed immediately upon a Council
 vote. If they plan on returning to active participation soon, they will be
-given a grace period of one year. If they don’t return to active participation
+given a grace period of six months. If they don’t return to active participation
 within that time period they will be removed by vote of the Council without
 further grace period. All former Council members can be considered for
-membership again at any time in the future, like any other Project Contributor.
+membership again at any time in the future, like any other Core Contributor.
  Retired Council members will be listed on the project website, acknowledging
 the period during which they were active in the Council.
 
 The Council reserves the right to eject current Members, other than the BDFL,
 if they are deemed to be actively harmful to the project’s well-being, and
 attempts at communication and conflict resolution have failed.
 
-### Conflict of interest
+<!-- todos:
 
-It is expected that the BDFL and Council Members will be employed at a wide
-range of companies, universities and non-profit organizations. Because of this,
-it is possible that Members will have conflict of interests. Such conflict of
-interests include, but are not limited to:
+- vote of no conficence. Either core or recurring contributors
+should also be able to remove people from the council, even if by a very high
+majority vote. IMO (Oriol) this point is not up for discussion in itself, only in
+how it should happen.
 
--   Financial interests, such as investments, employment or contracting work,
-    outside of The Project that may influence their work on The Project.
--   Access to proprietary information of their employer that could potentially
-    leak into their work with the Project.
+- election discussion. Do we want to keep the current council votes new council
+  or should core/recurring contributors vote?
```

Comment:
IMO there should be a size limit to the steering council.
If it's a small council, I think the core contributors should vote; otherwise with a big steering council it probably wouldn't make a big difference in practice?

Reply:
Agreed, smaller size is more efficient. And most decisions are made with the full core community anyway (order of dozens of people).

I'd be in favour of core community voting steering council, rather than steering council votes steering council (but not "simple" recurring contributors). But still current steering council can propose new council and express preference etc., of course.